Microsoft.Toolkit.Uwp.UI/Extensions/Tree/VisualTree.cs

```diff
@@ -156,5 +187,28 @@ public static T FindAscendant<T>(this DependencyObject element)
 
             return parent.FindAscendant<T>();
         }
+
+        /// <summary>
+        /// Find first visual ascendant control of a specified type.
+        /// </summary>
+        /// <param name="element">Child element.</param>
+        /// <param name="type">Type of ascendant to look for.</param>
+        /// <returns>Ascendant control or null if not found.</returns>
+        public static object FindAscendant(this DependencyObject element, Type type)
```

Comment:
Why is this needed? What's wrong with
`public static T FindAscendant<T>(this DependencyObject element)`

Reply:
See comment on FindDescendant.